diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 4937a8edcfb3981d3974d2a0fa9bafdb92cde131..50f1d080dd30c5a94a996cd70d7be3083f3a6e2a 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -68,7 +68,7 @@ exports.networks = [
     { chainId: 274n, shortName: 'lachain' },
     { chainId: 280n, shortName: 'zksync-goerli' },
     { chainId: 282n, shortName: 'zkTCRO' },
-    { chainId: 288n, shortName: 'boba' },
+    { chainId: 288n, shortName: 'Boba' },
     { chainId: 291n, shortName: 'orderly' },
     { chainId: 300n, shortName: 'zksync-sepolia' },
     { chainId: 314n, shortName: 'filecoin' },
@@ -204,7 +204,8 @@ exports.networks = [
     { chainId: 9000n, shortName: 'evmos-testnet' },
     { chainId: 9001n, shortName: 'evmos' },
     { chainId: 9700n, shortName: 'MainnetDev' },
-    { chainId: 9728n, shortName: 'boba-testnet' },
+    { chainId: 9728n, shortName: 'BobaBnbTestnet' },
+    { chainId: 28882n, shortName: 'BobaSepolia' },
     { chainId: 10000n, shortName: 'smartbch' },
     { chainId: 10001n, shortName: 'smartbchtest' },
     { chainId: 10081n, shortName: 'joct' },
@@ -266,7 +267,7 @@ exports.networks = [
     { chainId: 48900n, shortName: 'zircuit-mainnet' },
     { chainId: 53457n, shortName: 'dodochain' },
     { chainId: 54211n, shortName: 'islmt' },
-    { chainId: 56288n, shortName: 'boba-bnb' },
+    { chainId: 56288n, shortName: 'BobaBnb' },
     { chainId: 57000n, shortName: 'tsys-rollux' },
     { chainId: 57054n, shortName: 'blaze' },
     { chainId: 57073n, shortName: 'ink' },
