diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 2a60ac262584a6cdae0dbdc3e73058be914c0f6c..0b62401bc19508c17ef8d706244d6f510cd16148 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -193,7 +193,8 @@ exports.networks = [
     { chainId: 9000n, shortName: 'evmos-testnet' },
     { chainId: 9001n, shortName: 'evmos' },
     { chainId: 9700n, shortName: 'MainnetDev' },
-    { chainId: 9728n, shortName: 'boba-testnet' },
+    { chainId: 9728n, shortName: 'BobaBnbTestnet' },
+    { chainId: 28882n, shortName: 'BobaSepolia' },
     { chainId: 10000n, shortName: 'smartbch' },
     { chainId: 10001n, shortName: 'smartbchtest' },
     { chainId: 10081n, shortName: 'joct' },
@@ -253,7 +254,7 @@ exports.networks = [
     { chainId: 48900n, shortName: 'zircuit-mainnet' },
     { chainId: 53457n, shortName: 'dodochain' },
     { chainId: 54211n, shortName: 'islmt' },
-    { chainId: 56288n, shortName: 'boba-bnb' },
+    { chainId: 56288n, shortName: 'BobaBnb' },
     { chainId: 57000n, shortName: 'tsys-rollux' },
     { chainId: 57054n, shortName: 'blaze' },
     { chainId: 57073n, shortName: 'ink' },
