diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 00ae69d..9f5cb2f 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -60,7 +60,7 @@ exports.networks = [
     { chainId: 255n, shortName: 'kroma' },
     { chainId: 280n, shortName: 'zksync-goerli' },
     { chainId: 282n, shortName: 'zkTCRO' },
-    { chainId: 288n, shortName: 'boba' },
+    { chainId: 288n, shortName: 'Boba' },
     { chainId: 291n, shortName: 'orderly' },
     { chainId: 300n, shortName: 'ogn' },
     { chainId: 314n, shortName: 'filecoin' },
@@ -170,7 +170,8 @@ exports.networks = [
     { chainId: 8822n, shortName: 'iotaevm' },
     { chainId: 9000n, shortName: 'evmos-testnet' },
     { chainId: 9001n, shortName: 'evmos' },
-    { chainId: 9728n, shortName: 'boba-testnet' },
+    { chainId: 9728n, shortName: 'BobaBnbTestnet' },
+    { chainId: 28882n, shortName: 'BobaSepolia' },
     { chainId: 10000n, shortName: 'smartbch' },
     { chainId: 10001n, shortName: 'smartbchtest' },
     { chainId: 10081n, shortName: 'joct' },
@@ -222,7 +223,7 @@ exports.networks = [
     { chainId: 48900n, shortName: 'zircuit-mainnet' },
     { chainId: 53457n, shortName: 'dodochain' },
     { chainId: 54211n, shortName: 'islmt' },
-    { chainId: 56288n, shortName: 'boba-bnb' },
+    { chainId: 56288n, shortName: 'BobaBnb' },
     { chainId: 57000n, shortName: 'tsys-rollux' },
     { chainId: 58008n, shortName: 'sepPGN' },
     { chainId: 59140n, shortName: 'linea-goerli' },
